nixos/modules/services/networking/wireguard.nix

```diff
@@ -153,35 +182,43 @@ let
   wgCommand = "${pkgs.wireguard}/bin/wg";
 
   generateUnit = name: values:
+    assert (values.privateKey != null || values.privateKeyFile != null);
```

Comment:
Actually I don't - this just checks that at least one of the two is set. But you're right, I should.